decko-rails/lib/decko/engine.rb

```diff
@@ -27,10 +26,11 @@ def gem_root
 
   end
 
-  class Decko::Engine < Rails::Engine
+  class Engine < ::Rails::Engine
     
     paths.add "app/controllers", :eager_load => true
     paths.add 'gem-assets',      :with => 'public/assets'
+    paths.add 'lib/tasks',       :with => 'lib/decko/tasks', :glob => '**/*.rake'
```

Comment:
I didn't actually mean to add this.  It shouldn't hurt, right?  Unless it throws errors when the directory isn't there, but I doubt it does that.